Cargo.toml

```diff
@@ -23,44 +23,55 @@ name = "is_enabled"
 bench = false
 name = "dump-features"
 path = "src/bin/dump-features.rs"
+required-features = ["async-std", "surf-client"]
 
 [dependencies]
 anyhow = "1.0.44"
 arc-swap = "1.4.0"
-async-std = "1.10.0"
+async-std = { version = "1.10.0", optional = true }
 enum-map = "2.0.1"
-futures = "0.3.17"
 futures-timer = "3.0.2"
 hostname = "0.3.1"
-http-client = "6.5.1"
-http-types = "2.12.0"
 ipnet = "2.3.1"
 log = "0.4.14"
-maplit = "1.0.2"
 murmur3 = "0.5.1"
 rand = "0.8.4"
 serde_json = "1.0.68"
 serde_plain = "1.0.0"
-surf = "2.3.1"
+surf = { version = "2.3.1", optional = true }
 
 [dependencies.chrono]
 features = ["serde"]
 version = "0.4.19"
 
+[dependencies.reqwest]
+default-features = false
+features = ["json"]
+version = "0.11.10"
+optional = true
+
 [dependencies.serde]
 features = ["derive"]
 version = "1.0"
 
 [dev-dependencies]
 criterion = "0.3.5"
+futures = "0.3.17"
+maplit = "1.0.2"
 num_cpus = "1.13.0"
 simple_logger = "2.1.0"
 
 [dev-dependencies.tokio]
-features = ["rt-multi-thread", "macros"]
+features = ["rt-multi-thread", "macros", "time"]
 version = "1.12.0"
 
 [features]
+default = ["surf-client"]
+reqwest-client = ["reqwest", "reqwest/default-tls"]
```

Comment:
Nitpicking, but maybe a better name would be `reqwest-client-native-tls`?

Reply:
perhaps but I'm not sure it matters, and as a principle, I prefer less typing except when it really matters.